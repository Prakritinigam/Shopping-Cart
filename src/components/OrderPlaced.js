import withHOC from "./withHOC";

function OrderPlaced() {
    return <h2>Thank you, your orders have been placed.</h2>;
}

export default withHOC(OrderPlaced);