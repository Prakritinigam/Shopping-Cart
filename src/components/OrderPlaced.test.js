import { render } from '@testing-library/react';
import OrderPlaced from './OrderPlaced';
import React from 'react';

it('renders', () => {
  const { asFragment } = render(<OrderPlaced/>);
  expect(asFragment()).toMatchSnapshot();
});