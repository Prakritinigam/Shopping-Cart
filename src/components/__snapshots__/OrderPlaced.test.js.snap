// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <div
    style="text-align: center;"
  >
    <h2>
      Thank you, your orders have been placed.
    </h2>
    <a
      href="/"
    >
      Shopping Cart
    </a>
  </div>
</DocumentFragment>
`;
